qt/scientific_interfaces/Inelastic/Analysis/FunctionBrowser/MultiFunctionTemplateModel.cpp

```diff
@@ -226,24 +226,24 @@ void MultiFunctionTemplateModel::setParameter(ParamID name, double value) {
   }
 }
 
-boost::optional<double> MultiFunctionTemplateModel::getParameter(ParamID name) const {
+std::optional<double> MultiFunctionTemplateModel::getParameter(ParamID name) const {
   auto const paramName = getParameterName(name);
-  return paramName ? m_model->getParameter(*paramName) : boost::optional<double>();
+  return paramName ? m_model->getParameter(*paramName) : std::optional<double>();
```

Comment:
Is this initialization `std::optional<double>` equivalent to `std::nullopt`?

Reply:
Good question, no I believe `std::optional<double>` instantiates the default value for double, which is 0.0. I think this isn't very clear so I'll change it!

Reply:
Actually, I think these should probably return nullopt, so good spot

Reply:
I changed my mind again, I think some of these functions should explicitly return the default value for either double or std::string, and the `getParameterName` function should return a std::nullopt. This is based on what they are being used for in the rest of the model